src/app/public/plugin-resources/code-examples/inline-form/repeaters/inline-form-demo.component.html

```diff
@@ -0,0 +1,76 @@
+<sky-repeater>
+  <sky-repeater-item *ngFor="let item of items"
+    [inlineFormConfig]="inlineFormConfig"
+    [inlineFormTemplate]="inlineFormTemplate"
+    [showInlineForm]="activeInlineFormId === item.id"
+    (inlineFormClose)="onInlineFormClose($event)"
+  >
+
+    <sky-repeater-item-title>
+      <div
+        class="pull-left"
```

Comment:
These `pull-` classes are using CSS floats... I don't think we want to suggest that our consumers use them. Could we show this example without using floats?

Reply:
These are hold-overs from skyux2 (and the current code example). I'll replace them with something else.